spec/javascripts/components/story/expanded_story/expanded_story_owned_by.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ExpandedStoryOwnedBy from 'components/story/ExpandedStory/ExpandedStoryOwnedBy';
+
+describe('<ExpandedStoryOwnedBy />', () => {
+  const setup = propOverrides => {
+    const defaultProps = {
+      users: [
+        { id: 1, name: 'foo' },
+        { id: 2, name: 'bar' }
+      ],
+      story: { _editing: { ownedById: '' } },
+      onEdit: sinon.spy(),
+      ...propOverrides
+    };
+
+    const wrapper = shallow(<ExpandedStoryOwnedBy {...defaultProps} />);
+    const input = wrapper.find('input');
+
+    return { wrapper, input };
+  };
+
+  it('renders properly', () => {
+    const { wrapper } = setup();
+
+    expect(wrapper).toExist();
+  });
+
+  describe('input element', () => {
+    it('has a empty string as default value when ownedById is null', () => {
```

Comment:
You could add a test when ownedById has a value. 
Something like:

```js
it('has value of ownedById when ownedById is not empty', () => {
  const ownedById = 'some value'
  const story = { _editing: { ownedById } }

  const { input } = setup({ story }, ownedById);

  expect(input).toHaveValue(ownedById);
});
```

To this works 👆 , you have to update `setup` method:
```js
const setup = (propOverrides, ownedById = '') => {
  const defaultProps = {
    users: [
      { id: 1, name: 'foo' },
      { id: 2, name: 'bar' }
    ],
    story: { _editing: { ownedById } },
    onEdit: sinon.spy(),
    ...propOverrides
  };

  const wrapper = shallow(<ExpandedStoryOwnedBy {...defaultProps} />);
  const input = wrapper.find('input');

  return { wrapper, input };
};
```

Reply:
Thanks :D, but i removed this in another component, and I test if there's a userId or not!